tst/EnergyPlus/unit/EMSManager.unit.cc

```diff
@@ -826,6 +827,416 @@ TEST_F( EnergyPlusFixture, EMSManager_CheckIfAnyEMS_OutEMS ) {
+		index = 42 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR42" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 998.2331862652, 0.00000001 ); // RhoH2O 60.0 =
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 43 ).Operator, FuncSevereWarnEp );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 43 ).NumOperands, 1 );
+		index = 43 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR43" );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 44 ).Operator, FuncWarnEp );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 44 ).NumOperands, 1 );
+		index = 44 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR44" );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 45 ).Operator, FuncTrendValue );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 45 ).NumOperands, 2 );
+		index = 45 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR45" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.0, 0.00000001 ); // TrendValue 0.5 1.5 = ???
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 46 ).Operator, FuncTrendAverage );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 46 ).NumOperands, 2 );
+		index = 46 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR46" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.0, 0.00000001 ); // TrendAverage 0.5 1.5 = ???
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 47 ).Operator, FuncTrendMax );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 47 ).NumOperands, 2 );
+		index = 47 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR47" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.0, 0.00000001 ); // TrendMax 0.5 1.5 = ???
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 48 ).Operator, FuncTrendMin );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 48 ).NumOperands, 2 );
+		index = 48 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR48" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.0, 0.00000001 ); // TrendMin 0.5 1.5 = ???
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 49 ).Operator, FuncTrendDirection );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 49 ).NumOperands, 2 );
+		index = 49 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR49" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.0, 0.00000001 ); // TrendDirection 0.5 1.5 = ???
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 50 ).Operator, FuncTrendSum );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 50 ).NumOperands, 2 );
+		index = 50 + offset;
+		EXPECT_EQ( DataRuntimeLanguage::ErlVariable( index ).Name, "VAR50" );
+		EXPECT_NEAR( DataRuntimeLanguage::ErlVariable( index ).Value.Number, 0.0, 0.00000001 ); // TrendSum 0.5 1.5 = ???
```

Comment:
@EnergyArchmage the previous 6 tests return 0 as a result for @Trend functions. Either I'm doing something wrong or there is an issue here. I assume it's me.